wifiphisher/common/opmode.py

```diff
@@ -154,6 +154,13 @@ def _check_args(self, args):
                         '] --deauth-channels (-dC) requires channels in range 1-14.'
                     )
 
+        # if both args.mitminterface and args.internetinterface are provided, the
+        # former takes precedence and the latter gets overwritten.
+        if args.mitminterface and args.internetinterface:
+            print(('[' + constants.O + '!' + constants.W +
+                  '] using  both --mitminterface (-mI) and --internetinterface (-iI)'
+                  ' is redundant. Ignoring --internetinterface (-iI).'))
+
```

Comment:
This will always appear when using `-m`. See below. 

In addition, all sentences should start with a capital letter (e.g. "Using" instead of "using").